Parsers/ASimWebSession/Parsers/ASimWebSessionBarracudaWAF.yaml

```diff
@@ -0,0 +1,305 @@
+          EventVendor = "Barracuda",
+          status_code = toint(HTTPStatus_s)
+      | extend
+          EventResult_TR = case(
+                      status_code between (200 .. 299),
+                      "Success", 
+                      status_code between (400 .. 599),
+                      "Failure",
+                      status_code between (300 .. 399),
+                      "Partial",
+                      "NA"
+                  ),
+          RuleName = RuleType_s
+      | extend
+          Dvc = UnitName_s,
+          EventResult = iff(LogType_s == "TR", EventResult_TR, EventResult_WF),
+          EventStartTime = unixtime_milliseconds_todatetime(tolong(DeviceReceiptTime_s)),
+          DstIpAddr = ServerIP_s,
+          SrcIpAddr = ClientIP_s,
+          DstBytes = tolong(BytesReceived_d),
+          DstPortNumber = toint(ServerPort_d),
+          HttpCookie = Cookie_s,
+          HttpReferrer = Referer_s,
+          HttpRequestBodyBytes = tolong(BytesReceived_d),
+          HttpRequestMethod = Method_s,
+          HttpResponseBodyBytes = tolong(BytesSent_d),
+          NetworkDuration = toint(TimeTaken_d),
+          HttpUserAgent = UserAgent_s,
+          NetworkSessionId = SessionID_s,
+          Rule = RuleName,
+          SrcPortNumber = toint(ClientPort_d),
+          SrcUsername = CertificateUser_s,
+          DstUsername = AuthenticatedUser_s,
+          Url = URL_s,
+          HttpResponseCacheControl = iff(
+                                CacheHit_d == 0,
+                                "Response from the server",
+                                "Response from the cache"
+                            ),
+          AdditionalFields = bag_pack(
+                        "ProxyIP",
+                        ProxyIP_s,
+                        "ProxyPort",
+                        ProxyPort_d
+                    ),
+          DvcHostname = host_s,
+          DvcIpAddr = HostIP_s,
+          EventResultDetails = HTTPStatus_s
+      | extend 
+          SrcUsernameType = iff(isnotempty(SrcUsername), "Simple", ""),
+          DstUsernameType = iff(isnotempty(DstUsername), "Simple", "")
+      | extend
+          Duration = NetworkDuration,
+          SessionId = NetworkSessionId,
+          EventEndTime = EventStartTime,
+          UserAgent = HttpUserAgent,
+          User = SrcUsername,
+          IpAddr = SrcIpAddr,
+          Src = SrcIpAddr,
+          HttpStatusCode = EventResultDetails
+      | project-away
+          *_d,
+          *_s,
+          _ResourceId,
+          severity,
+          status_code,
+          RawData,
+          EventResult_*,
+          SourceIP,
+          Message,
+          EventType_lookup,
+          Computer,
+          MG,
+          ManagementGroupName,
+          TenantId,
+          SourceSystem
+      )
+  };
+  let BarracudaCEF = (disabled: bool=false) {
+      CommonSecurityLog
+      | where not(disabled) and DeviceVendor == "Barracuda" and DeviceProduct == "WAF"
```

Comment:
Can we confirm on the filters. In the sample present in the github, I see different values.
https://github.com/Azure/Azure-Sentinel/blob/master/Sample%20Data/CEF/barracudaWAFCEF.txt

If both are possible, we should include both like
DeviceVendor startswith "Barracuda"

Reply:
We check and both can be possible. So we updated the condition such a way both should be consider.